src/module.cpp

```diff
@@ -14,8 +14,12 @@ static void init(v8::Local<v8::Object> target) {
     // expose hello_async method
     Nan::SetMethod(target, "hello_async", standalone_async::hello_async);
 
-    // expose hello_world class
-    HelloWorld::Init(target);
+    // expose HelloObject class
+    object_sync::HelloObject::Init(target);
+
+    // Notice there are multiple "hello" functions as part of this module:
+    // 1) standalone::hello
+    // 2) HelloObject.hello()...not really sure why this function doesnt have to be expliclty exposed
```

Comment:
@springmeyer curious why `HelloObject.hello()` doesnt have to be expliclty exposed in the same way as the other methods. Is it because it's handled as part of the class exposure?